packages/liferay-npm-scripts/test/scripts/webpack.js

```diff
@@ -31,31 +31,51 @@ describe('scripts/webpack.js', () => {
 
 	it('invokes webpack', () => {
 		webpack();
-		expect(spawnSync).toHaveBeenCalledWith('webpack', []);
+
+		expect(spawnSync.mock.calls).toHaveLength(1);
+		expect(spawnSync.mock.calls[0]).toHaveLength(2);
+		expect(spawnSync.mock.calls[0][0]).toBe('webpack');
+		expect(spawnSync.mock.calls[0][1]).toHaveLength(2);
+		expect(spawnSync.mock.calls[0][1][0]).toBe('--config');
+		expect(spawnSync.mock.calls[0][1][1]).toMatch(/webpack.config.js$/);
```

Comment:
All of this expansion seems like overkill that harms readability/writability to guard against a bug that will never happen. So you know there was exactly one call now, but do you *really* think that we'd accidentally call it twice?

Reply:
We can remove the `toHaveLength` expectations, but the problem was with the last argument (the `toMatch` one) that cannot be tested with `toHaveBeenCalledWith` because it is an absolute path into the developer's machine :-(.

If you can think of any other way to test this with a cleaner syntax I would be glad to change it because it looks horrible to me too :sweat_smile: .

Reply:
I think you can test it with `toHaveBeenCalledWith`:

```javascript
expect(spawnSync).toHaveBeenCalledWith(
    'webpack', [
        '--config',
        expect.stringMatching(/\bwebpack\\.config\\.js$/,        
    ]
);	
```

There's also [`expect.stringContaining`](https://jestjs.io/docs/en/expect#expectnotstringcontainingstring) if you don't want the rigor of the regular expression.

Reply:
:-o :clap: